src/lib/fields.ts

```diff
@@ -95,8 +95,15 @@ async function updatePullRequestLinkOnRecord(
 }
 
 async function getOrLinkPullRequestRecord(pr: PullRequest) {
+  // The PR might be a webhook or a GQL object. Handle both cases.
+  const head = (("headRef" in pr && pr.headRef) ||
+    ("head" in pr && pr.head)) as any;
+  const headRefName = head?.ref || head?.name || "";
```

Comment:
Suggestion for this:

```
  // The PR might be a webhook or a GQL object. Handle both cases.
  let headRefName: string | undefined;

  if ("headRef" in pr) {
    headRefName = pr.headRef?.name;
  } else {
    headRefName = pr.head?.ref;
  }
```

I found all the || a bit confusing and there are only 2 cases, the pr is a webhook or a gql object, so branch on that. what do you think? gets rid of the any too (edited)